src/metabase/api/search.clj

```diff
@@ -39,6 +35,9 @@
    (s/optional-key :offset-int)  (s/maybe s/Int)})
 
 (def ^:private SearchableModel
+  (apply s/enum search-config/all-models))
+
+(def ^:private DBModel
   (apply s/enum search-config/searchable-models))
```

Comment:
this name has now turned pretty confusing, if SearchableModels is not searchable-models